sql-plugin/src/main/spark324/scala/org/apache/spark/sql/rapids/shims/RapidsHadoopWriterUtils.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019-2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*** spark-rapids-shim-json-lines
+{"spark": "324"}
+{"spark": "332"}
+{"spark": "333"}
+spark-rapids-shim-json-lines ***/
+package org.apache.spark.sql.rapids.shims
+
+import org.apache.hadoop.mapred.JobID
+
+import org.apache.spark.internal.io.SparkHadoopWriterUtils
+
+/* This shim object uses the underlying fix from SparkHadoopWriterUtils in case this
+   logic is updated in the future for other versions of Spark 3.3 */
```

Comment:
Style nit: Multi-line comments either should use `//` at the start of each line or put a `*` at the start of each line, e.g.:
```suggestion
/**
 * This shim object uses the underlying fix from SparkHadoopWriterUtils in case this
 * logic is updated in the future for other versions of Spark 3.3
 */
```